pylib/cqlshlib/helptopics.py

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CASSANDRA_NEW_DOC_FORMAT = False
```

Comment:
Perhaps use dedicated classes for new and old formats and switch on object creation in `cql3handling.py`.

Reply:
Thx for looking!

Right. But I was forward thinking to having only the new docs available, either both locally and online or only online. Where I would just have to do a quick regexp replace instead, so a new class etc didn't come to mind as I was seeing this as an interim.